VimeoNetworking/Sources/NSError+Extensions.swift

```diff
@@ -83,7 +83,7 @@ public extension NSError
      
      - returns: a new `NSError`
      */
-    class func error(withDomain domain: String?, code: Int?, description: String?) -> NSError
+    @nonobjc class func error(withDomain domain: String?, code: Int?, description: String?) -> NSError
```

Comment:
Aren't things `@nonobjc` by default? Do we need to add this ever?

Reply:
You would think, right? It looks like Foundation subclasses like this require adding this attribute. Here's the compiler error I was getting:

```
NSError+Extensions.swift:86:16: Method cannot be in an @objc extension 
of a class (without @nonobjc) because the type of the parameter 2 cannot
be represented in Objective-C
```

Reply:
Ah gotcha